sdk/core/azure-core/src/main/java/com/azure/core/http/rest/ResponseConstructorsCache.java

```diff
@@ -68,6 +65,10 @@ private MethodHandle locateResponseConstructor(Class<?> responseClass) {
         try {
             lookupToUse = ReflectionUtilsApi.INSTANCE.getLookupToUse(responseClass);
         } catch (Throwable t) {
+            if (t instanceof Error) {
+                throw (Error) t;
+            }
```

Comment:
it seems that ReflectionUtilsApi is in the implementation package. Maybe we could change declaration there to be more specific ?

Reply:
I changed ReflectionUtilsApi to throw Exception, looks like I didn't update this.